### ISSUE TITLE
Unhandled exception on opening IPH

### ISSUE THREAD

**Issue by Zumochi** [EVEIPH/EVE-IPH#9]
Windows 7 when trying to start IPH.

`Message: An error occurred creating the form. See Exception.InnerException for details.  The error is: SQLite error near "AND": syntax error`

More extensive log from the logfile:

```
8/8/2015 4:27:12 PM, System.InvalidOperationException: An error occurred creating the form. See Exception.InnerException for details.  The error is: SQLite error
near "AND": syntax error ---> System.Data.SQLite.SQLiteException: SQLite error
near "AND": syntax error
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteCommand.GetStatement(Int32 index)
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader()
   at EVE_Isk_per_Hour.IndustryFacility.LoadFacility(FacilitySettings SearchFacilitySettings, Boolean FacilityDefault)
   at EVE_Isk_per_Hour.frmMain.SetAllFacilities(Boolean LoadDefault)
   at EVE_Isk_per_Hour.frmMain..ctor()
   --- End of inner exception stack trace ---
   at EVE_Isk_per_Hour.My.MyProject.MyForms.Create__Instance__[T](T Instance)
   at EVE_Isk_per_Hour.My.MyProject.MyForms.get_frmMain()
   at EVE_Isk_per_Hour.My.MyApplication.OnCreateMainForm()
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.OnRun()
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.DoApplicationModel()
```

Re-trying several times results in the same problem. I may or may not have pressed scroll lock while it was loading (idk figured it's worth mentioning, switched to that button as push to talk key yesterday).

Is my database broken?

**Comment by Zifrian** [comment]
This is a known error but I'm having issues tracking it down. Can you zip your 'Settings' folder with all the xml files in it and send it to ZifrianEVE@gmail.com? Thanks

**Comment by Zumochi** [comment]
For further info, I tried deleting all files except the ones in the Settings folder and re-"installing" them from the binary zip.

**Comment by tylo** [comment]
I too am having this issue. At first I tried updating the program, as I hadn't played Eve in about 3 months. It was failing to connect to anything, so I decided to uninstall and did a fresh reinstall.

After that, it was connecting for updates, did an update, and then started having the same exact error Zumochi is saying.

I am on Windows 8.1.

**Comment by Zifrian** [comment]
I found the issue. You need to delete the BPManufacturing settings Xml file in the settings. I hope to get a fix up soon.

**Comment by Zumochi** [comment]
Awesome! :+1:

**Comment by Zifrian** [comment]
This should be fixed in Build 3.1.5732.39083. Added error handling to catch this instead of crashing.

**Comment by Zumochi** [comment]
Updated to the latest version successfully. However now I am getting [this error message](http://i.imgur.com/9EJlT2z.png) on starting. I can guess what's going on and how to fix it, if only I knew what the "Unknown facility" is.

**Comment by Zifrian** [comment]
This is what was causing the crash you posted about earlier. You have a facility saved with 'none' for the name or type. I try to get the best information I can on that but it's difficult at the moment. Either check your files (It's probably the BPManufacturingFacilitySettings.xml file) or re-save the facilities as you use them. I'll look into providing better feedback but right now I just wanted the program to open without crashing. I still can't figure out how someone could save the facility with None. I think it was due to an error I fixed that allowed users to save a POS without a system.
